src/content/content_instance.cc

```diff
@@ -756,20 +755,21 @@ void ContentItem::print(void) {
 #endif
 
 void ContentInstance::HandleScanFileRequest(const picojson::value& msg) {
-  if (msg.contains(STR_CONTENT_URI)) {
```

Comment:
I don't see any functional change here. Removing indentation levels does make sense, and I'd certainly support writing it this way if you were adding the code, but changing it just for aesthetic purposes isn't very useful and makes looking at the commit history more complicated.